### PR TITLE
Basically complete cclcc delusion trigger

### PR DESCRIPTION
**Pull request by KKhanhH**
CCLCC Delusion Triggers+Renderer changes so the drawspriteoffset will rotate around the offset.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -430,17 +433,18 @@ set(Impacto_Header
         src/games/chlcc/systemmenu.h
         src/games/chlcc/trophymenu.h
         src/games/chlcc/delusiontrigger.h
-
+
```
Comment (MrRevo3D) [inline CMakeLists.txt]: Stray whitespace changes looks like

File: src/hud/delusiontrigger.h [hunk 2]
```diff
@@ -15,8 +15,11 @@ class DelusionTriggerBase {
   DelusionTriggerBase(int& delusionState, UiState showState);
   virtual void Update(float dt) = 0;
   virtual void Render() = 0;
-  virtual void Show() = 0;
   virtual void Hide() = 0;
+  virtual void Show(){};
+  virtual bool Show(int param1, int param2, int param3) { return true; };
```
Comment (MrRevo3D) [inline src/hud/delusiontrigger.h]: Missed renaming vars here and below

**Review by MrRevo3D — COMMENTED** [review]
Just these 2 things and we're good to go for now

File: CMakeLists.txt (hunk 1 above)
Comment (KKhanhH) [reply CMakeLists.txt]: isn't this one fine? it separates chlcc and cc

Comment (KKhanhH) [reply CMakeLists.txt]: oh never mind added a tab

**Pull request merged**